src/main/interface/msp.c

```diff
@@ -2235,7 +2235,7 @@ static mspResult_e mspCommonProcessInCommand(uint8_t cmdMSP, sbuf_t *src)
                     osdConfigMutable()->enabled_stats[addr] = value;
                 } else if (addr < OSD_ITEM_COUNT) {
                     /* Set element positions */
-                    osdConfigMutable()->item_pos[addr] = value;
+                    /* osdConfigMutable()->item_pos[addr] = value; */
```

Comment:
Should this be commented out?

Reply:
No, this was just temporary while what to do with MSP was discussed.